packages/xod-client/src/editor/containers/Patch/index.jsx

```diff
@@ -139,6 +140,11 @@ class Patch extends React.Component {
         className={cn('PatchWrapper-container', currentMode)}
         ref={(r) => { this.dropTargetRootRef = r; }}
       >
+        <ReactResizeDetector
+          handleWidth
+          handleHeight
+          onResize={this.props.actions.patchWorkareaResized}
```

Comment:
It's spamming with actions on window resize.
Maybe better to add debounce here?